packages/web/src/components/host-remix-contest-modal/HostRemixContestModal.tsx

```diff
@@ -33,18 +36,34 @@ const messages = {
   hostTitle: 'Host Remix Contest',
   description:
     'Turn your track into a remix challenge and co-sign your favorite submissions.',
-  deadlineTitle: 'Submission Deadline',
   hint: 'You can host one contest per song and adjust the submission deadline anytime within 90 days of the contest start.',
   startContest: 'Start Contest',
   save: 'Save',
   contestEndDateLabel: 'Last day to submit to contest',
   endDateError: 'Contest end date must be in the future within 90 days',
```

Comment:
can we move all these messages to common and reuse them in web and mobile?

Reply:
Yea good thought. I'll do this with the mobile changes PR